api/core/v1alpha1/ticdc_types.go

```diff
@@ -0,0 +1,187 @@
+// Copyright 2024 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
```

Comment:
a new api group for ticdc ?

Reply:
I prefer to use the same API group as TiDB, as if we delete a cluster the TiCDC should be deleted; it should be easier to reuse other common tasks if we use the same API group?